### PR TITLE
added supported environs, reorganized FTP ext doc

### PR DESCRIPTION
**Pull request by anaxceron**
Signed-off-by: anaxceron <ana.ceron@broadcom.com>

Please review the content I have added/edited:
- Broke up [Using Zowe Explorer](https://docs.zowe.org/stable/user-guide/ze-usage) into sections for better UX
- Edited the (Zowe Explorer) **Sample Use Cases** section to reflect correct VS Code UI terminology and standardize instructions/format
- Added **Supported operating systems, environments, and platforms** to **Using Zowe Explorer** section
- Broke up [Using Zowe Explorer FTP Extension](https://docs.zowe.org/stable/user-guide/ze-ftp) into sections for better UX and made edits to reflect correct VS Code UI terminology and standardize instructions/format
- Made minor edits to [CICS Extension/Providing feedback and contributing](https://docs.zowe.org/stable/user-guide/ze-provide-feedback-contribute) so it matches similar content for FTP Extension

Once reviewed/approved, I will do any additional edits and merge the PR once I return to the office on **Thurs., 10/20**.

### CODE REVIEW

**Review by zFernand0 — APPROVED** [review]
my apologies for taking so long to review this.
I believe the changes look great! 
And thanks for adding the environments 😋 
Zowe Explorer may need to implement some changes to fully support Node.js 18 (which is becoming LTS pretty soon) 😋

File: docs/user-guide/ze-ftp-using-ze-ftp-ext.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+3. Select a profile to display it in the **Side Bar**.
+
+4. Hover over the profile and click the **Search** icon.
+
+5. Enter the applicable values in the **picker** field:
+    - For data sets, select or enter the data set name.
+    - For USS, select or enter the path.
+    - For jobs, select or enter the job owner and job prefix.
+
+## Creating an FTP profile with Zowe Explorer
```
Comment (JillieBeanSim) [inline docs/user-guide/ze-ftp-using-ze-ftp-ext.md:35]: @zFernand0 with create profile actions should we be promoting use of v2?

Comment (zFernand0) [reply docs/user-guide/ze-ftp-using-ze-ftp-ext.md:35]: I agree we should 😋 
But maybe it require it's own section in `Using Zowe Explorer` that the FTP section can link to (and possibly expand)
I believe this is just content that was there from before 😋

Comment (anaxceron) [reply docs/user-guide/ze-ftp-using-ze-ftp-ext.md:35]: @zFernand0 @JillieBeanSim  I agree to work on this on a separate PR

**Review by JillieBeanSim — APPROVED** [review]

**Pull request merged**